roxie/ccd/ccdserver.cpp

```diff
@@ -2917,19 +3030,35 @@ class CRoxieServerMultiInputBaseActivity : public CRoxieServerActivity
 
 class CRoxieServerMultiInputActivity : public CRoxieServerMultiInputBaseActivity
 {
+    bool startInputsSequentially = false;
 public:
     CRoxieServerMultiInputActivity(IRoxieAgentContext *_ctx, const IRoxieServerActivityFactory *_factory, IProbeManager *_probeManager, unsigned _numInputs)
         : CRoxieServerMultiInputBaseActivity(_ctx, _factory, _probeManager, _numInputs)
     {
+        startInputsSequentially = factory->startInputsSequentially() || factory->queryQueryFactory().queryOptions().startInputsSequentially;
```

Comment:
see comment above.  It may not even be worth caching in the activity - the overhead of calling the virtual is fairly low.